tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -1589,8 +1592,17 @@ const CounterIdAliasPair *DeclResultIdMapper::getCounterIdAliasPair(
     if (counter != fieldCounterVars.end())
```

Comment:
question: do you know why we use SmallVector<4> for indices?  I see OpAccessChain also has like 4 indices, while spec doesn't seem to mention a limit.

Reply:
For this particular situation, the number of indices rarely exceeds 4. Even if it did, it wouldn't be by a lot, and SmallVector would handle it, just slightly inefficiently.

Reply:
Thanks!